pkg/defaults/defaults_freebsd.go

```diff
@@ -0,0 +1,51 @@
+//go:build freebsd
+// +build freebsd
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package defaults
+
+import (
+	gocni "github.com/containerd/go-cni"
+)
+
+const AppArmorProfileName = "nerdctl-default"
```

Comment:
This should be empty.

Eventually this variable should be only referred from `*_linux.go`, though.